coretex/entities/task_run/task_run.py

```diff
@@ -591,7 +591,8 @@ def runLocal(
         saveSnapshot: bool,
         name: Optional[str],
         description: Optional[str] = None,
-        parameters: Optional[List[Dict[str, Any]]] = None
+        parameters: Optional[List[Dict[str, Any]]] = None,
+        entryPoint: Optional[str] = None
```

Comment:
Missing in documentation

Reply:
it is not a name of the script, it is relative path to the script inside of the project